test/spmd/test_dynamo_spmd.py

```diff
@@ -205,6 +205,8 @@ def test_dynamo_spmd_mark_sharding_outside_of_compile(self):
     dynamo_res = dynamo_linear(xla_x)
     self.assertEqual(met.metric_data('CompileTime')[0], compile_count)
 
+  # https://github.com/pytorch/xla/pull/6921#issuecomment-2062106737
+  @unittest.skip("Failing in CI")
   def test_mark_sharding_inside_compile(self):
```

Comment:
@wonjoolee95 should we just remove this test? It should not pass from first place since we call the regualr `mark_sharding`